components/datetime/src/date.rs

```diff
@@ -156,6 +176,9 @@ impl FromStr for MockDateTime {
     }
 }
 
+/// This macro defines a struct for each type of unit to be used in a DateTime. Each
+/// unit is bounded by a range. The traits implemented here will return a Result on
+/// whether or not the unit is in range from the given input.
```

Comment:
The line breaks here are weird. Can you realign?

Reply:
Can you clarify what the line-break style should look like? I was struggling to understand it. In other projects I've done hard cut offs at a column width, so things flow where they happen to end. I was noticing that a lot of the docs have very long line widths. What is the cut-off strategy?

Reply:
Ah, no, retracting. I'm not sure why Github showed me this comment malformed. It looks good now.